src/applications/simple-forms/40-0247/tests/e2e/0247-pmc.cypress.spec.js

```diff
@@ -109,16 +111,18 @@ const testConfig = createTestConfig(
           cy.get('@testData').then(data => {
             const { additionalAddress, additionalCopies } = data;
 
-            cy.get(v3StepHeaderSelector)
+            cy.get(v3StepHeaderSelector, { timeout: 10000 })
               .should('be.visible')
               .then(() => {
-                cy.get('input[name="root_additionalAddress_state"]')
+                cy.get('input[name="root_additionalAddress_state"]', {
+                  timeout: 10000,
+                })
                   .should('not.be.disabled')
                   .then(() => {
                     // callback to avoid field-disabled errors, but
                     // even now we must wait a bit!
                     // eslint-disable-next-line cypress/no-unnecessary-waiting
-                    cy.wait(500);
+                    cy.wait(1000);
                     fillAddressWebComponentPattern(
                       'additionalAddress',
                       additionalAddress,
```

Comment:
can you remove the newly introduced `skip: Cypress.env('CI'),` below to make sure this is properly testing your changes?

Reply:
Re-enabled.  thx for catching that, Robert!